tests/arp/test_stress_arp.py

```diff
@@ -161,7 +166,12 @@ def test_ipv6_nd(duthost, ptfhost, config_facts, tbinfo, ip_and_intf_info,
         try:
             add_nd(ptfadapter, ip_and_intf_info, ptf_intf_index, nd_avaliable)
             if not skip_traffic_test:
-                pytest_assert(wait_until(20, 1, 0, lambda: get_fdb_dynamic_mac_count(duthost) >= nd_avaliable),
+                # There is a certain probability of hash collision, we set the percentage as 1% here
```

Comment:
hi @yutongzhang-microsoft  could we have this number shows up in the summary when it get failed? It would be easier for person who looks into this failure.

Reply:
It will shows in test logs.